src/sentry/issues/occurrence_consumer.py

```diff
@@ -319,6 +328,28 @@ def process_occurrence_message(
         txn.set_tag("result", "dropped_feature_disabled")
         return None
 
+    rate_limit_enabled = options.get("issues.occurrence-consumer.rate-limit.enabled")
+    if rate_limit_enabled:
+        rate_limit_key = create_rate_limit_key(occurrence_data["fingerprint"][0])
+        rate_limit_quota = Quota(**options.get("issues.occurrence-consumer.rate-limit.quota"))
```

Comment:
would write a quick unit test to makesure the quota is initialized with the correct values from the option (can do outside of a UnitTest class, just something like

```
def test_occurrence_rate_limit_quota():
   rate_limit_quota = Quota(**options.get("issues.occurrence-consumer.rate-limit.quota"))
   assert rate_limit_quota.blahblah == blabhbalh
````